shared/js/smart-screen/scrollable.js

```diff
@@ -203,7 +203,6 @@
       // Determine initial focus (depends on reversed or not)
       var initNode = this.getItemFromNode(
                      this.getNode(this._isReversed ? this.length - 1 : 0));
-      this.silentFocus(initNode);
 
```

Comment:
Using initNode to calculate the translateX is enough. We don't have to silent focus initNode here.
Doing so would cause:
(1) Original focused item lost focus
(2) Since this is a silent focus, it would not trigger style update. So on the UI, the original focused item is still in the focused style but, under the scrollable the current focused item has became the initNode.